tests/flow/test_rdb_compat.py

```
@@ -0,0 +1,275 @@
+from random import randint, seed
+from common import *
+
+FALKORDB_C_IMAGE = 'falkordb/falkordb:v4.18.1'
+
+# ──────────────────────────── Docker helpers ────────────────────────────
+
+def run_db(image):
+    """Start a FalkorDB container on a random port."""
+    import docker
+    client = docker.from_env()
+    port = randint(49152, 65535)
```

Ruff (0.15.9)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)